src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://127.0.0.1:3306/test?user=root";
+    private static final String DB_USER = "root";
+    private static final String DB_PASSWORD = "Gx42Cbj1!";
+    private static final String DRIVER_CLASS = "com.mysql.cj.jdbc.Driver";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", DB_USER);
+        DB_PROPERTIES.put("password", DB_PASSWORD);
+
+        try {
+            Class.forName(DRIVER_CLASS);
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException("Can't load MySQL driver",e);
+        }
+    }
+
+    public static Connection getConnection() throws SQLException {
```

Comment:
The getConnection() method throws an SQLException. It would be better to handle this exception in the method and throw a custom exception instead. This can improve error handling and make the code cleaner.